tests/jwt-cli.rs

```diff
@@ -662,13 +663,33 @@ mod tests {
         assert_eq!(expected, key);
     }
 
+    #[test]
+    fn encoding_key_from_secret_handles_base64() {
+        let b64 = "+t0vs/PPB0dvyYKIk1DYvz5WyCUds5DLy07ycOK5oHA=";
+        let mut arg: String = "b64:".to_owned();
+        arg.push_str(b64);
```

Comment:
I think using `format!` might be a little easier to read here. This way you avoid mutability and it's already a `String` type 😁 

```suggestion
        let arg = format!("b64:{}", b64);
```

Reply:
Makes sense to me... I'm such a Rust newbie!  😁